commit/plugin_e2e_test.go

```diff
@@ -438,11 +438,11 @@ func TestPlugin_E2E_AllNodesAgree_ChainFee(t *testing.T) {
 	nodes := make([]ocr3types.ReportingPlugin[[]byte], len(oracleIDs))
```

Comment:
changing the tests to simulate real results as we shouldn't expect fees for the destChain itself.